XamlControlsGallery/Navigation/NavigationRootPage.xaml

```diff
@@ -104,10 +105,12 @@
             Header=" "
             HeaderTemplate="{StaticResource NavigationViewHeaderTemplate}"
             IsTabStop="False"
-            PaneOpening="NavigationViewControl_PaneOpened"
+            IsTitleBarAutoPaddingEnabled="False"
+            PaneOpening="NavigationViewControl_PaneOpening"
             PaneClosing="NavigationViewControl_PaneClosing"
             DisplayModeChanged="NavigationViewControl_DisplayModeChanged"
-            SelectionChanged="OnNavigationViewSelectionChanged">
+            ItemInvoked="OnNavigationViewItemInvoked"

```

Comment:
Why using `ItemInvoked` rather than `SelectionChanged` ?

Reply:
Good callout.  Fixed (here and in main branch)